stm/vernac_classifier.ml

```diff
@@ -202,7 +202,7 @@ let classify_vernac e =
       let poly = List.fold_left (fun poly f ->
           match f with
           | VernacPolymorphic b -> b
-          | (VernacProgram | VernacLocal _) -> poly
+          | (VernacProgram | VernacLocal _ | VernacGuarded _) -> poly
```

Comment:
This is not going to scale if we get many attributes. You should just use `_ ->` here.